brightwind/utils/utils.py

```diff
@@ -252,3 +254,39 @@ def validate_json(json_to_check, schema):
             print(f"Failed schema part: {error.get('schema_path')}\n")
     
     return data_is_valid
+
+
+def apply_scale_factor(data : Union[float, pd.Series, np.array],
+                       scale_factor: Union[float, int]):
+    """
+    Scales data by the scale_factor.
+
+    :param value:           Value to scale by the scale_factor
+    :type value:            float or pandas.Series or numpy.array
+    :param scale_factor:    Scale factor as a float.
+    :type scale_factor:     float
+    :returns:               scale_factor times value
+    :rtype:                 float or pandas.Series or numpy.array depending on type(value)
+
+        **Example usage**
+    ::
+    import brightwind as bw
+
+    # scale float by scale_factor of 0.5
+    bw.apply_scale_factor(3, 0.5)
+    # 1.5
+
+    # scale np.array by scale_factor of 0.5
+    bw.apply_scale_factor(np.array([0, 1, 2]), 0.5)
+    # [0, 0.5, 1]
+    
+    # scale pd.Series by scale_factor of -10
+    bw.apply_scale_factor(pd.Series([10, 20, 30, 40]), -10)
+    # [-100, -200, -300, -400]
+    """
+
+    if not isinstance(data, (float, int, pd.Series, np.ndarray)):
```

Comment:
if accepts int then you should add this type also to function and docstring type definitions and error sentence